### PR TITLE
ebs-br: unify command into existing command

### PR DESCRIPTION
**Pull request by D3Hunter**
### What problem does this PR solve?

Issue Number: close #xxx

Problem Summary:

### What is changed and how it works?
- unify `backup ebs` into `backup full`
- unify `restore ebs/data` into `restore full`

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: br/pkg/config/ebs.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ type EBSStore struct {
 // pd cluster id/alloc id, cluster resolved ts and tikv configuration.
 type ClusterInfo struct {
 	Version    string            `json:"cluster_version" toml:"cluster_version"`
-	MaxAllocID uint64            `json:"max_alloc_id" toml:"max_alloc_id"`
+	FullBRType string            `json:"full_br_type" toml:"full_br_type"`
```
Comment (3pointer) [inline br/pkg/config/ebs.go:60]: maybe change it to `full_backup_type`?

File: br/cmd/br/backup.go [hunk 2]
```diff
@@ -31,6 +31,15 @@ func runBackupCommand(command *cobra.Command, cmdName string) error {
 		ctx, store = trace.TracerStartSpan(ctx)
 		defer trace.TracerFinishSpan(ctx, store)
 	}
+
+	if cfg.FullBackupType == task.FullBackupTypeEBS {
+		if err := task.RunBackupEBS(ctx, tidbGlue, &cfg); err != nil {
+			log.Error("failed to backup", zap.Error(err))
```
Comment (fengou1) [inline br/cmd/br/backup.go:37]: a little concern here, zap.Error(err) may print a call stack and above return errors.Trace(err) may also print in caller. can we remove it?

File: br/pkg/task/backup_ebs.go [hunk 3]
```diff
@@ -219,7 +186,7 @@ func RunBackupEBS(c context.Context, g glue.Glue, cmdName string, cfg *BackupEBS
 	// Step.2 starts call ebs snapshot api to back up volume data.
 	// NOTE: we should start snapshot in specify order.
 
-	progress := g.StartProgress(ctx, cmdName, int64(storeCount), !cfg.LogProgress)
+	progress := g.StartProgress(ctx, "ebs backup", int64(storeCount), !cfg.LogProgress)
```
Comment (fengou1) [inline br/pkg/task/backup_ebs.go:189]: what about snapshot backup? since we will also move to other platform e.g gcs.

**Review by fengou1 — COMMENTED** [review]
rest LGTM

File: br/pkg/task/backup_ebs.go (hunk 3 above)
Comment (D3Hunter) [reply br/pkg/task/backup_ebs.go:189]: change to `backup`

**Review by 3pointer — APPROVED** [review]
LGTM

**Pull request merged**